packages/devtools-source-map/src/worker.js

```diff
@@ -3,29 +3,21 @@ const {
   getGeneratedLocation,
   getOriginalLocation,
   getOriginalSourceText,
+  hasMappedSource,
   applySourceMap,
   clearSourceMaps
 } = require("./source-map");
 
+const { workerUtils: { workerHandler }} = require("devtools-utils");
 
 // The interface is implemented in source-map to be
 // easier to unit test.
-const publicInterface = {
```

Comment:
Jest has made it much easier to write unit tests, which means we can likely bring back the worker methods into this file.

I didn't want to complicate this PR too much though